Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py

```diff
@@ -0,0 +1,20 @@
+import demistomock as demisto
```

Comment:
why you need this?

Reply:
It's in every content script, this is the template (it's actually removed when the automation deployed)